### ISSUE TITLE
dungeon signs remain signs

### ISSUE THREAD

**Issue by somrune** [DRE2N/DungeonsXL#201]
I have a problem that started earlier this week.
Whenever I enter the dungeon through a portal or with the command, the signs are still there and no mobs or events spawn, could this be due to a wrongly added sign or perhaps because I don't use a dungeon config file?
This started when I tried to add classes and it didn't work, eventhough the YAML parsers say my files are correct.

My console also doesn't give any error, so I don't think it's a bug. Could be another plugin interfering with dungeons? i have tried reinstalling the plugin but that did not fix the issue

**Comment by Sataniel98** [comment]
Did you add lobby, ready and start signs? Most signs are initialized when the game starts, so not before you trigger the ready sign.

**Comment by somrune** [comment]
Yes I have, but once I'm inside the signs dont work, even the [Leave] sign in the lobby wont let me leave the dungeon, nor does the command, it lets me leave the party but I'm still in the dungeon then.
The [ready] sign can't be clicked and signs can even be broken that are inside the dungeon.

When I try to edit the dungeon, I often get the error  "you have to join a group a first" both entry and exit.
This all started when I tried to add classes to the dungeon, but removing them has not solved my issues thus far.

I will try another reinstall tomorrow though, hoping it's because I did something wrong and can fix it.

**Comment by somrune** [comment]
I removed everything, including the dungeon maps (with backups obviously) and discovered that whenever I typed /dxl edit a new edit file was created, but it was never deleted after leaving the dungeon, same with attempting to play the dungeon, I had 5-7 of each in my worlds folder after deleting the plugin and reloading, those files were still there and recognized as seperate worlds.
this might be the reason the signs and commands didn't work properly.

After deleting those, reinstalling the plugin and reuploading my dungeon maps, everything seems to work again.
